ampl-converter/src/main/java/com/powsybl/ampl/converter/AmplConstants.java

```diff
@@ -9,14 +9,40 @@
 import java.util.Locale;
 
 /**
- *
  * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
  */
 public final class AmplConstants {
 
     private AmplConstants() {
     }
 
+    public static final String XNODE_COUNTRY_NAME = "XNODE";
+    public static final String FAULT = "fault";
+    public static final String DESCRIPTION = "description";
+    public static final String SUBSTATION = "substation";
+    public static final String TARGET_V = "targetV (pu)";
+    public static final String TARGET_Q = "targetQ (MVar)";
```

Comment:
I think that MVAr spelling is generally better than MVar

Reply:
We usually use `MVar` in the PowSyBl code.

Could you revert the change?
(Note that for now, the files are not consistent because `getBusesColumns()` still uses `MVar`).